src/templates/common/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  transformBreadcrumbs,
+  deriveLastBreadcrumbFromPath,
+  deriveLcBreadcrumbs,
+  filterInvalidCrumbs,
+} from '@/lib/utils/breadcrumbUtils'
+import { BreadcrumbItem } from '@/types/dataTypes/drupal/field_type'
+
+interface BreadcrumbProps {
+  breadcrumbs?: BreadcrumbItem[]
+  breadcrumbTitle?: string
+  disableAnalytics?: boolean
+  deriveBreadcrumbsFromUrl?: boolean
+  replaceLastItem?: boolean
+  constructLcBreadcrumbs?: boolean
+  lcBreadcrumbsTitleInclude?: boolean
+  hideHomeBreadcrumb?: boolean
+  customCrumbHomeText?: string
+  entityPath?: string
+}
+const Breadcrumbs = ({
+  breadcrumbs,
+  entityPath,
+  breadcrumbTitle,
+  disableAnalytics,
+  deriveBreadcrumbsFromUrl,
+  replaceLastItem,
+  constructLcBreadcrumbs,
+  lcBreadcrumbsTitleInclude,
+  hideHomeBreadcrumb,
+  customCrumbHomeText,
+}: BreadcrumbProps) => {
+  if (!breadcrumbs) return null
+
+  if (customCrumbHomeText) {
+    breadcrumbs = breadcrumbs.map((crumb) => {
+      if (crumb.title === 'Home') {
+        return { ...crumb, title: customCrumbHomeText }
+      }
+      return crumb
+    })
+  } else if (hideHomeBreadcrumb) {
+    breadcrumbs = breadcrumbs.filter((crumb) => crumb.title !== 'Home')
+  }
+
+  if (deriveBreadcrumbsFromUrl) {
+    breadcrumbs = deriveLastBreadcrumbFromPath(
+      breadcrumbs,
+      breadcrumbTitle,
+      entityPath,
+      replaceLastItem
+    )
+  }
+
+  if (constructLcBreadcrumbs) {
+    breadcrumbs = deriveLcBreadcrumbs(
+      breadcrumbs,
+      breadcrumbTitle,
+      entityPath,
+      lcBreadcrumbsTitleInclude
+    )
+  }
+
+  const breadcrumbList = transformBreadcrumbs(breadcrumbs)
+  const filteredCrumbs = filterInvalidCrumbs(breadcrumbList)
+
+  return (
+    <va-breadcrumbs
+      disable-analytics={disableAnalytics}
+      class="hydrated va-nav-breadcrumbs"
+      wrapping={false}
+    >
+      {filteredCrumbs.map((crumb, index) => {
+        return (
+          <a key={index} href={crumb.href}>
+            {crumb.label}
+          </a>
+        )
+      })}
+    </va-breadcrumbs>
+  )
+}
+
+Breadcrumbs.defaultProps = {
+  label: 'Breadcrumbs',
+  uswds: true,
+  string: '',
+}
```

Comment:
**SHOULD**

Is this needed? It looks like it is a remnant of copying something as a starting point. Can it be removed?

FWIW, I think it's more idiomatic to define default prop values in the function signature (when destructuring `props`).

Reply:
Yeah this was leftover from previous iteration, removed